app/views/planning_applications/review/immunity_details/_form.html.erb

```diff
@@ -17,42 +17,29 @@
         <%= form.govuk_radio_button(
               :action,
               "accepted",
-              checked: @review_immunity_detail.accepted? && !@review_immunity_detail.reviewed_at.nil?,
-              label: {text: "Accept"},
-              disabled: !editable
+              checked: @planning_application.immunity_detail.current_evidence_review_immunity_detail.accepted? && !@planning_application.immunity_detail.current_evidence_review_immunity_detail.reviewed_at.nil?,
+              label: {text: "Accept"}
             ) %>
         <%= form.govuk_radio_button(
               :action,
               "rejected",
-              checked: @review_immunity_detail.rejected? && !@review_immunity_detail.reviewed_at.nil?,
-              label: {text: "Return to officer with comment"},
-              disabled: !editable
+              checked: @planning_application.immunity_detail.current_evidence_review_immunity_detail.rejected? && !@planning_application.immunity_detail.current_evidence_review_immunity_detail.reviewed_at.nil?,
+              label: {text: "Return to officer"}
```

Comment:
Maybe these `checked` values could be extracted into variables? They're a bit complex to read in the template like this.